.github/workflows/backend.yml

```diff
@@ -41,6 +42,7 @@ jobs:
         flags: back-end
 
   be-linter-clj-kondo:
+    if: github.event.pull_request.draft == false
```

Comment:
@metabase/backend-developers does it help you if we leave `kondo` and/or `eastwood` linters running when a PR is still in a draft mode or do you usually have some pre-commit hooks already set up locally?

Reply:
Kondo is pretty fast I don't think it hurts anything to leave it on for a draft PR TBH. We're not paying extra for it right?